Cyra/lab2/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+cmake_minimum_required(VERSION 3.14)
+project(nixintro)
+include_directories(include)
+
+
+add_library(hellolib SHARED src/hellolib.cpp)
```

Comment:
this doesnt appear to be the correctly named source file, cmake is case-sensitive for these file names